data.json

```diff
@@ -437,12 +437,6 @@
     "imageUrl": "https://user-images.githubusercontent.com/54971231/103836002-6bdb9f00-5090-11eb-9501-dc21ccbf3607.png",
     "issueNumber": 227
   },
-  {
```

Comment:
I'm not sure why this person's entry got removed.

Reply:
what should I do about this?

Reply:
You could paste the this entry back in.

Reply:
I have Pasted it, Thanks Allan